src/plugins/workspace/server/saved_objects/integration_tests/workspace_saved_objects_client_wrapper.test.ts

```diff
@@ -526,5 +672,48 @@ describe('WorkspaceSavedObjectsClientWrapper', () => {
       }
       expect(SavedObjectsErrorHelpers.isNotFoundError(error)).toBe(true);
     });
+
+    it('should be able to delete any data when user is dashboard admin', async () => {
+      const createResultOne = await repositoryKit.create(
+        internalSavedObjectsRepository,
+        'dashboard',
+        {},
+        {
+          permissions: {
+            read: { users: ['foo'] },
+            write: { users: ['foo'] },
+          },
+        }
+      );
+
+      await dashboardAdminSavedObjectedClient.delete('dashboard', createResultOne.id);
+
+      let errorOne;
```

Comment:
We should not use such kind of variable names.

Reply:
I had corrected it.